src/ebddocx2table/docxtableconverter.py

```diff
@@ -263,6 +266,74 @@ def _handle_single_table(
                     )
                 )
 
+    # see above boolean_outcome and subsequent_step_number could be ignored iff schemes of *-numbers are always the same
+    # pylint:disable=too-many-locals
+    def _handle_single_table_star_exception(
+        self,
+        table: Table,
+        multi_step_instructions: List[MultiStepInstruction],
+        row_offset: int,
+        rows: List[EbdTableRow],
+        row_index: int,
+    ) -> None:
+        """
+        Completes table when handling of single table (out of possible multiple tables for 1 EBD) hit a step
+        with several instructions. Those instructions will be split in individual steps.
+        As above, the results are written into rows, sub_rows and multi_step_instructions. Those will be modified.
+        """
+        use_cases: List[str] = []
+        # first init
+        enhanced_table_row = self._enhance_list_view(table=table, row_offset=row_offset)[row_index]
+        use_cases = _get_use_cases(enhanced_table_row.cells)
+        star_case_result_code = (
+            enhanced_table_row.cells[len(use_cases) + self._column_index_result_code].text.strip() or None
+        )
+        star_case_note = enhanced_table_row.cells[len(use_cases) + self._column_index_note].text.strip() or None
+        while row_index < len(self._enhance_list_view(table=table, row_offset=row_offset)):
+            enhanced_table_row = self._enhance_list_view(table=table, row_offset=row_offset)[row_index]
+            step_number = str(int(rows[-1].step_number) + 1)
```

Comment:
```suggestion
            step_number = str(int(last(rows).step_number) + 1)
```
find ich lesbarer. ist aber nicht wichtig

Reply:
1b287fa6e6e82b73ae5c37d4aebee3373b58f4a8